src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -42,4 +44,15 @@ def filter_by_involvement(filter_involvement)
       User.session.outgoing_requests
     end
   end
+
+  def filter_by_involvement_for_project(filter_by_involvement, project)
+    case filter_by_involvement
+    when 'all'
+      BsRequest.with_actions.where('bs_request_actions.target_project = (?) OR bs_request_actions.source_project = (?)', project.name, project.name)
+    when 'incoming'
+      OpenRequestsFinder.new(BsRequest, project.name).incoming_requests(project.open_requests.values.sum)
+    when 'outgoing'
+      OpenRequestsFinder.new(BsRequest, project.name).outgoing_requests(project.open_requests.values.sum)
```

Comment:
We should note down to allow filtering by any state, we only display open requests now